third_party/payjs/third_party/random_uuid/Random.uuid.js

```diff
@@ -0,0 +1,94 @@
+/** @license
+Math.uuid.js (v1.4)
+http://www.broofa.com
+mailto:robert@broofa.com
+Copyright (c) 2010 Robert Kieffer
+Dual licensed under the MIT and GPL licenses.
```

Comment:
Looks like this code snippet is from here: https://www.broofa.com/2008/09/javascript-uuid-function/

Which was updated in: https://github.com/broofa/node-uuid

And then closed in favor of: https://github.com/kelektiv/node-uuid

@kunal-rp Who maintains `payjs` internally? Can they comment on how important it is to use a well-designed UUID for Google transaction ID?

Reply:
wendiz@ can provide more context for the library usage decision; he's OOO, but I pinged him to this question

Reply:
This is also the same version payjs used through our internal library. This library has served our transaction id generation so far. And from the payjs code size perspective I'd prefer us not to switch to a later version with more code unless it's necessary. @choumx let me know if you have any particular concern about this. Thanks!

Reply:
Fair enough. 

One related point: we avoid recursive `third_party` folders by only including a bundled/compiled version of the library. Can we do this for `payjs`? 

I tried to bazel build it via the README instructions but hit an error:
```
Starting local Bazel server and connecting to it...
ERROR: /Users/willchou/payjs/WORKSPACE:1:1: name 'http_archive' is not defined
ERROR: Error evaluating WORKSPACE file
ERROR: error loading package '': Encountered error while reading extension file 'closure/private/java_import_external.bzl': no such package '@io_bazel_rules_closure//closure/private': error loading package 'external': Could not load //external package
ERROR: error loading package '': Encountered error while reading extension file 'closure/private/java_import_external.bzl': no such package '@io_bazel_rules_closure//closure/private': error loading package 'external': Could not load //external package
INFO: Elapsed time: 2.110s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
```

Reply:
So if it's okay to add the library in compiled format we can do that.

Kunal can you take a look at the build error when you get a chance?

Reply:
@kunal-rp  Were you able to dig into the build error described above?

Reply:
for clarity : the current aog extensions use a modified version of payjs; that code will be a third party library referenced in the payment service specified to only aog extensions. Thus, the public payjs is obselete

Reply:
To clarify on "bundling":

> New third_party libs should only be a single binary (bundle), rather than having its own directory structure (e.g. a nested third_party folder). 
> 
> E.g. `third_party/aog-payjs/payjs.js` should have no imports, and its dependencies should be inlined (bundled) directly into the same file.
> 
> This is typically done with tools like rollup, webpack or browserify. There's already precedence for this in `third_party/subscriptions-project/swg.js`, which uses rollup.

Reply:
@choumx 

added bundled file, renamed to payjs.js using rollup

also rm all third party files